### PR TITLE
fix: crash ofter logout during incoming call SQSERVICES-1902

### PR DESCRIPTION
**Pull request by MarcinR**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/SQSERVICES-1902" title="SQSERVICES-1902" target="_blank">BugSQSERVICES-1902</a>  [iOS] App crashes while on the login page for some time
  </td></table>
  <br />
 

#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [x] contains a reference JIRA issue number like `SQPIT-764`
  - [x] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [x] is free of optional paragraphs and you have filled the relevant parts to the best of your ability

### CODE REVIEW

File: Wire-iOS/Sources/UserInterface/Settings/CellDescriptors/SettingsSignOutCellDescriptor.swift [hunk 1]
```diff
@@ -41,6 +42,8 @@ final class SettingsSignOutCellDescriptor: SettingsExternalScreenCellDescriptor
         if selfUser.usesCompanyLogin || password != nil {
             weak var topMostViewController: SpinnerCapableViewController? = UIApplication.shared.topmostViewController(onlyFullScreen: false) as? SpinnerCapableViewController
             topMostViewController?.isLoadingViewVisible = true
+            AVSMediaManager.sharedInstance()?.stop(sound: .ringingFromThemInCallSound)
+            AVSMediaManager.sharedInstance()?.stop(sound: .ringingFromThemSound)
```
Comment (KaterinaWire) [inline Wire-iOS/Sources/UserInterface/Settings/CellDescriptors/SettingsSignOutCellDescriptor.swift:46]: Are these related changes?

Comment (MarcinR) [reply Wire-iOS/Sources/UserInterface/Settings/CellDescriptors/SettingsSignOutCellDescriptor.swift:46]: yes, we have to disable calling sound when signing out or else it'll still playing on login view

Comment (KaterinaWire) [reply Wire-iOS/Sources/UserInterface/Settings/CellDescriptors/SettingsSignOutCellDescriptor.swift:46]: I assume that the same changes you did for the release branch, correct?

**Review by KaterinaWire — APPROVED** [review]
